code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -598,11 +598,10 @@
 		hand_index = (active_hand_index % held_items.len)+1
 	var/obj/item/held_item = get_active_held_item()
 	if(held_item)
-		if(istype(held_item, /obj/item/twohanded))
-			var/obj/item/twohanded/T = held_item
-			if(T.wielded == 1)
-				to_chat(usr, "<span class='warning'>Your other hand is too busy holding [T].</span>")
-				return
+		var/datum/component/two_handed/comp_twohand = held_item.GetComponent(/datum/component/two_handed)
```

Comment:
you're using get component here

Reply:
That i am, this is not the item, this is not a standard location check. and offer som other option or this stays.

Reply:
use some method to let the component communicate to the mob that that hand is also taken.

Reply:
This has been redesigned and is now signal based with support from the component